### ISSUE TITLE
Cannot set sensitivity other than int * 100

### ISSUE THREAD

**Issue by 7HE-W0R1D** [z3ntu/RazerGenie#168]
I used to set the sensitivity of my Razer Viper V2 Pro to 650 DPI. However, this number seems not possible in RazerGenie. I cannot input numbers in the field, it seems to be only a indicator of the current sens. And the slider is only able to set sensitivities of integer * 100 (400, 500, 600, 700). I know 650 is possible because I set it once by editing some config under Linux.

Did I miss anything? Is this function available in RazerGenie?

Thanks in advance!

**Comment by z3ntu** [comment]
Yes, I've recently also noticed this. When I initially made the UI I thought 100-DPI increments would be enough but I'm planning to reduce the increments to 50, so you can select e.g. 650 DPI.

Is this enough or do you also want some option to e.g. also select 653 DPI so in 1 DPI increments? The hardware does support this (or at least we can send it to the device), but I don't really think there's a use case for this.

**Comment by dodoradio** [comment]
could it be made so that you can set any integer dpi in the text box, while limiting to 50 dpi increments on the slider?

**Comment by z3ntu** [comment]
Do you have a use case for that?

**Comment by 7HE-W0R1D** [comment]
> Do you have a use case for that?

I've been having some busy time and didn't have a chance to follow, but a use case for @dodoradio suggestion would be:

For general users, the 50 increment will allow for quick adjustment to the sens they want (normally int * 100), they are also not sensitive to sens changes so 50 will be about right if they want to change and test a new sens, they are not looking for the accurate numbers.

For gamers / advanced users, they know what sensitivity they want and they know what the number means. So its okay to give them the largest degree of freedom to set the sensitivity.

These are just my initial thoughts :)

**Comment by z3ntu** [comment]
This should be fixed via cf2d79c and earlier commits and will be included in the next release!
